src/page-modules/departures/layout.tsx

```diff
@@ -40,39 +73,69 @@ function DeparturesLayout({ children }: DeparturesLayoutProps) {
 
   return (
     <div className={style.departuresPage}>
-      <div className={style.searchWrapper}>
-        <form className={style.searchContainer} onSubmit={onSubmitHandler}>
-          <div className={style.searchInput}>
-            <p className={style.searchInputLabel}>
+      <form className={style.container} onSubmit={onSubmitHandler}>
+        <div className={style.main}>
+          <div className={style.input}>
+            <Typo.p textType="body__primary--bold" className={style.heading}>
               {t(PageText.Departures.search.input.label)}
-            </p>
+            </Typo.p>
 
             <Search
               label={t(PageText.Departures.search.input.from)}
               onChange={setSelectedFeature}
             />
           </div>
 
-          <div className={style.searchDate}>
-            <p className={style.searchInputLabel}>
+          <div className={style.date}>
+            <Typo.p textType="body__primary--bold" className={style.heading}>
               {t(PageText.Departures.search.date.label)}
-            </p>
+            </Typo.p>
 
             <DepartureDateSelector
               initialState={departureDate}
               onChange={setDepartureDate}
             />
           </div>
+        </div>
+
+        <div className={style.buttons}>
+          <Button
+            title={t(PageText.Departures.search.buttons.alternatives.title)}
+            className={style.button}
+            mode={showAlternatives ? 'interactive_3' : 'interactive_2'}
```

Comment:
The Button component doesn't support the mode used in Figma. Should we add it, and if so what should the mode be called?

![2023-10-12 at 23 13 53](https://github.com/AtB-AS/planner-web/assets/14045515/45ff4aae-8fbe-4eda-a70e-da73dcafe29b)

Reply:
Yes, let's add it later.